index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <section class="Mobile_background">
```

Comment:
- [ ] In this project you don't need a section wrapping the whole document. For semantic purposes and according to your requirements, you need to start with `header` and after that a section, please delete this `section` tag and use it properly. There are even better ways to structure a document, for example, one that you could use is the following one:

``` html
<body>
  <header>
    <nav> 
      ...
    </nav>
  </header>

  <main>
    <section>
      ...
    </section>

    <section>
      <article>
        ...
      </article>
      <article>
        ...
      </article>
      <article>
        ...
      </article>
    </section>
  </main>
  <footer>
    ...
  </footer>
</body>
```

Reply:
When I put properties of background wanted on body{} in CSS, it doesn't t work, that s why I added a section tag for the all body content. I need guidelines, thank you